samuroi/util/mask_generator/basic_image_processing.py

```diff
@@ -0,0 +1,33 @@
+__author__ = 'stephenlenzi'
+
+import numpy as np
```

Comment:
I think this file is obsolete. it gets imported nowhere, and the two bottom functions are trivial. it the smooth_data is used somewhere then move it there and remote the basic_image_processing file